tests/schema/tests.py

```diff
@@ -1671,6 +1731,60 @@ class Meta:
         with self.assertRaises(IntegrityError):
             Tag.objects.create(title='bar', slug='foo')
 
+    def test_remove_field_uniqueness_does_not_remove_custom_constraints(self):
```

Comment:
`test_removing_field_constraint_does_not_remove_meta_constraint`? (Similar for other tests?)

Reply:
sorry, look like I miss point of you questions, by the way all tests in PR pretty similar